AutomatedTesting/Levels/Blast/Blast_ActorSplitsAfterCollision/Blast_ActorSplitsAfterCollision.prefab

```diff
@@ -0,0 +1,255 @@
+{
+    "ContainerEntity": {
+        "Id": "ContainerEntity",
+        "Name": "Blast_ActorSplitsAfterCollision",
+        "Components": {
+            "Component_[10182366347512475253]": {
+                "$type": "EditorPrefabComponent",
+                "Id": 10182366347512475253
```

Comment:
In all the prefabs you are adding newly in this PR, can you try removing all these required-editor components that only have default values? If the levels are loaded correctly, then all of these will be added to the in-memory template by default. So, I don't see a need for us to unnecessarily bloat these files with default components.

Reply:
I feel like that we can have a ticket to remove redundant components from prefab test level files since the first priority is to migrate automated tests from slice to prefab. 

Refine the prefab test level files is nice-to-have.

Reply:
Yes but if the tests pass after deleting the default required components, it'll be an easy-win. You don't have to do the double work again. Removing these components and running the tests shouldn't take that long. I'm not suggesting fixing any problems that come if you remove the components. I'm suggesting to simply remove the components and see if the tests still pass.

Reply:
I need to fix the prefab system error messages issue first(#7431) since it could be a blocker. I would prefer to wait until  I fix that first,  and then go back to try to delete each component on each prefab test levels to see if things broken or not. 

Also, if we try to modify the prefab file manually right now we might not be able to catch some issues caused by some prefab system changes interacting with the original test level files in the future.

Reply:
> prefab system changes interacting with the original test level files in the future.

Prefab system shouldn't interact with actual files except when loading and saving because everything happens in memory. Even if you remove these default components, they should be added back to the in-memory prefab template when loaded. So, there will not be any problems with feature parity loss whether the components are present in the file or not. Please add a follow up task to remove these default components

Reply:
Will do.

Reply:
Follow-up: #7505